tests/test_selectolax.py

```diff
@@ -0,0 +1,24 @@
+# ------------------------------------------------------------------
+# Copyright (c) 2024 PyInstaller Development Team.
+#
+# This file is distributed under the terms of the GNU General Public
+# License (version 2.0 or later).
+#
+# The full license is available in LICENSE, distributed with
+# this software.
+#
+# SPDX-License-Identifier: GPL-2.0-or-later
+# ------------------------------------------------------------------
+from PyInstaller.utils.tests import importorskip
+
+pytestmark = [
+    importorskip("tensorflow"),
+]
+
+
+def test_selectolax(pyi_builder):
```

Comment:
Please put this test at the end of `test_libraries.py` instead of its own file. 

And make sure to decorate `test_selectolax` with `@importorskip('selectolax')`.

Reply:
Addressed in 
- https://github.com/pyinstaller/pyinstaller-hooks-contrib/pull/841/commits/e7eeac877e5082a76c9225791d8e90a1bb8b01c4